scipio/src/parking.rs

```diff
@@ -58,13 +58,15 @@ impl Parker {
     pub(crate) fn new() -> Parker {
         // Ensure `Reactor` is initialized now to prevent it from being initialized in `Drop`.
         Parker {
-            inner: Rc::new(Inner {}),
+            inner: Rc::new(Inner {
+                wakers: RefCell::new(Vec::new()),
```

Comment:
See my extensive comment about this.
The vector is ultimately fine, but please initialize it with `with_capacity`.